### PR TITLE
authors: stub message for the old module

### PR DESCRIPTION
**Pull request by MSusik**
Signed-off-by: Mateusz Susik <mateusz.susik@cern.ch>

### CODE REVIEW

File: invenio/modules/authors/templates/authors/author_stub.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends 'page.html' %}
+
+{% block body %}
+
+    <div class="alert alert-warning">
+           You tried to access an author module page.
```
Comment (jirikuncar) [inline invenio/modules/authors/templates/authors/author_stub.html]: Please make the message translatable (`{{ _(...) }}`) and use 2 spaces for indentation. Thanks

File: invenio/modules/authors/templates/authors/author_stub.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2014 CERN.
```
Comment (jirikuncar) [inline invenio/modules/authors/templates/authors/author_stub.html]: wrong copyright year

File: invenio/modules/authors/templates/authors/author_stub.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends 'page.html' %}
```
Comment (jirikuncar) [inline invenio/modules/authors/templates/authors/author_stub.html]: use `{%- ... -%}` in order to avoid empty lines in output

**Comment by jirikuncar** [comment]
when you fix my comments you can add `Reviewed-by: me`.

**Comment by MSusik** [comment]
Fixed, tested, ready.

**Comment by MSusik** [comment]
Fixed, tested, ready.

**Pull request merged**